dev/bus/export/dev/bus.hpp

```diff
@@ -162,6 +162,11 @@ class generic_bus
     //!
     static err xfer(const bus_handler &handler);
 
+    //! \brief Returns a reference to underlying platform bus.
+    //! This allows to perform specific for underlying bus operations.
```

Comment:
Use \detail here, otherwise both lines will be merged into the brief description